### DISCUSSION TITLE
Ветка [Jun 12]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2431] [Discord Threads]
> 🙋‍♂️ **Sidney**  
##   
![image0](https://cdn.discordapp.com/attachments/661905455894888493/984750898817400893/unknown.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **alex.pro**  
#### coming soon 🙂 no one knows when  
#  
> 🙋‍♂️ **peter piper**  
#### are they some kind of permium-only offering?  
#  
> 👨‍🦲 **dstala**  
#### <@820992975298625547> No  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/984750899383656498)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
